pkg/operator/staticpod/controller/revision/deployment_controller_test.go

```diff
@@ -5,6 +5,7 @@ import (
 	"testing"
 	"time"
 
+	"github.com/openshift/library-go/pkg/operator/events"
```

Comment:
I think goland hates you :stuck_out_tongue:

Reply:
has nothing to do with goland... it is goimports weakness ;-)